build.gradle.kts

```diff
@@ -21,7 +21,7 @@ subprojects {
 
   extra.set("versions", mapOf(
       "opentelemetry" to "0.12.0",
-      "opentelemetryJavaagent" to "0.12.0-SNAPSHOT"
+      "opentelemetryJavaagent" to "0.13.0-SNAPSHOT"
```

Comment:
Do you need 0.13 snapshot or 0.12? The latter has been published at last: https://search.maven.org/search?q=g:io.opentelemetry.javaagent

Reply:
i currently don't need 0.13-SNAPSHOT, but probably we'll need it soon